tests/unit/test_row.py

```diff
@@ -359,6 +359,28 @@ def test_commit(self):
         row.commit()
         self.assertEqual(table.mutated_rows, [row])
 
+    @mock.patch("google.cloud.bigtable.table.Table.mutate_rows")
+    def test_commit_with_exception(self, mock_mutate_rows):
+        project_id = "project-id"
+        row_key = b"row_key"
+        table_name = "projects/more-stuff"
+        column_family_id = u"column_family_id"
+        column = b"column"
+
+        credentials = _make_credentials()
+        client = self._make_client(
+            project=project_id, credentials=credentials, admin=True
+        )
+        table = _Table(table_name, client=client)
+        row = self._make_one(row_key, table)
+        value = b"bytes-value"
+
+        # Perform the method and check the result.
+        row.set_cell(column_family_id, column, value)
+        from google.rpc import status_pb2
+        result = row.commit()
+        self.assertIsInstance(result, status_pb2.Status)
```

Comment:
Please test against the expected value from `_Table.mutate_rows` below, e.g.:
```python
        expected = status_pb2.Status(code=0)
        self.assertEqual(result, expected)
```

Reply:
Done.